src/hid_commands.cpp

```diff
@@ -328,6 +330,12 @@ bool interface::sync(const char*& result_begin, const char*& result_end)
     if (parsed) { // parsed frame, send ACK
       write(hid_, ACK);
       state_ = data;
+      if (result_begin != NULL) { // Message Terminator Record frame received, write CAN, done
```

Comment:
For some reason, my Contour Next USB returns "Software Error: Turn off then on. If error persists call Customer Service". The error is E86. Let me check it some more (couple of days?)...

Reply:
Hmm, maybe its because of the CAN at the end. Can you share the debug output?

Reply:
Indeed, it's the CAN at the end. I removed it and it works fine with the Contour Next. I'll check-in your changes and open a new issue to fix that.